crates/oxc_linter/src/snapshots/import_extensions.snap

```diff
@@ -0,0 +1,487 @@
+   ╭─[extensions.tsx:3:17]
+ 2 │                 import lib from "./bar.js";
+ 3 │                 import component from "./bar.jsx";
+   ·                 ──────────────────────────────────
+ 4 │                 import data from "./bar.json";
+   ╰────
+  help: Remove the file extension from this import.
+
+  ⚠ eslint-plugin-import(extensions): File extension "js" should not be included in the import declaration.
+   ╭─[extensions.tsx:2:17]
+ 1 │ 
+ 2 │                 import lib from "./bar.js";
+   ·                 ───────────────────────────
+ 3 │                 import component from "./bar.jsx";
+   ╰────
+  help: Remove the file extension from this import.
+
+  ⚠ eslint-plugin-import(extensions): File extension "json" should not be included in the import declaration.
+   ╭─[extensions.tsx:4:17]
+ 3 │                 import component from "./bar.jsx";
+ 4 │                 import data from "./bar.json";
+   ·                 ──────────────────────────────
+ 5 │             
+   ╰────
+  help: Remove the file extension from this import.
+
+  ⚠ eslint-plugin-import(extensions): File extension "jsx" should not be included in the import declaration.
+   ╭─[extensions.tsx:2:17]
+ 1 │ 
+ 2 │                 import component from "./bar.jsx";
+   ·                 ──────────────────────────────────
+ 3 │                 import data from "./bar.json";
+   ╰────
+  help: Remove the file extension from this import.
+
+  ⚠ eslint-plugin-import(extensions): File extension "json" should not be included in the import declaration.
+   ╭─[extensions.tsx:3:17]
+ 2 │                 import component from "./bar.jsx";
+ 3 │                 import data from "./bar.json";
+   ·                 ──────────────────────────────
+ 4 │             
+   ╰────
+  help: Remove the file extension from this import.
+
+  ⚠ eslint-plugin-import(extensions): File extension "coffee" should not be included in the import declaration.
+   ╭─[extensions.tsx:1:1]
+ 1 │ import "./bar.coffee"
+   · ─────────────────────
+   ╰────
+  help: Remove the file extension from this import.
+
+  ⚠ eslint-plugin-import(extensions): Missing file extension in import declaration
+   ╭─[extensions.tsx:4:17]
+ 3 │                 import barjson from "./bar.json";
+ 4 │                 import barnone from "./bar";
+   ·                 ────────────────────────────
+ 5 │             
+   ╰────
+  help: Add a file extension to this import.
+
+  ⚠ eslint-plugin-import(extensions): File extension "js" should not be included in the import declaration.
+   ╭─[extensions.tsx:2:17]
+ 1 │ 
+ 2 │                 import barjs from "./bar.js";
+   ·                 ─────────────────────────────
+ 3 │                 import barjson from "./bar.json";
+   ╰────
+  help: Remove the file extension from this import.
+
+  ⚠ eslint-plugin-import(extensions): File extension "" should not be included in the import declaration.
```

Comment:
diagnostic msg is incorrect, it should say "Missing file extension in import declaration"